src/acceptance-testing/release-readiness.md

```diff
@@ -0,0 +1,125 @@
+
+## Roles and Responsibilities
+
+While the Platforms team serves as the primary custodian of this release readiness process, its success relies on collaborative ownership with Protocol and contributions from across the organization.
+
+| Role | Responsibilities |
+|------|-----------------|
+| **Platforms Team** | • Maintain the Release Readiness Process (RRP)<br>• Run acceptance tests<br>• Make final pass/fail determinations<br>• Communicate test results to stakeholders |
+| **Feature Teams** | • Write acceptance tests for specific features<br>• Run feature-specific tests<br>• Fix identified issues in their features<br>• Provide input on severity of identified issues |
+| **Release Coordinator** | • Coordinate the overall release process<br>• Track progress against the Release Readiness Checklist<br>• Facilitate communication between teams<br>• Document release decisions and rationales |
+
+## Objectives
+
+The primary objectives of the Devnet Release Readiness process are:
+
+1. Release production networks without critical bugs
+2. Ensure high feature coverage through comprehensive testing
+3. Establish a clear process for devnet promotion decisions
+4. Provide visibility into the release readiness of devnets
+5. Define responsibilities for teams involved in the release process
+
+## Release Readiness Process
+
+### Prerequisites
+
+Before a devnet can be considered for release, the following prerequisites must be met:
+
+1. All required features must be deployed to the devnet
+2. A Release Readiness Checklist (RRC) must be defined for the devnet
+3. The [op-acceptor](https://github.com/ethereum-optimism/infra/tree/main/op-acceptor) testing framework must be configured for the devnet
+
+### Readiness Phases
+
+The devnets are expected to be live for short periods of time. For example, alphanets will be decomissioned after three weeks. This time will be roughly spent in five phases.
+
+#### 1. Initial Deployment
+
+- A devnet is deployed according to the [standard process](https://github.com/ethereum-optimism/devnets/blob/main/README.md)
+- The Platforms team (Infrastructure Pod) verifies that all requested features are properly deployed
+- Basic infrastructure checks are performed to ensure the network is operational
+- When all looks good, the devnet is made live and public
+
+#### 2. Public Usage
+
+- At this point the devnet is live; the general public is free to use & test with it
+- We collect any feedback
+
+#### 3. Acceptance Testing
+
+- All tests are executed against the [Release Readiness Checklist (RRC)](./release-checklist.md)
+- Automated acceptance tests are run using op-acceptor
+- For any manual tests remaining:
```

Comment:
We should include and probably even focus on exploratory testing in this phase - manual exploration and probing of the system looking for things that we previously missed rather than just ticking off predefined manual tests.

Reply:
Good idea; I think both would be good!
I've explicitly called out "exploratory testing" in 464b8de